starter-code/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Clue Game</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="styles.css" />
+    <script src="src/clue.js"></script>
+</head>
+<body>
+    <section class="card" id="character">
+        <h1>CHARACTER CARD</h1>
+        <h2 id="character-name" class="name"></h2>
+        <img id="character-pic" src="#" alt="character">
+        <p id="character-desc"></p>
+    </section>
```

Comment:
Perhaps `character` can be class too. So for your h2: 
```
<h2 class="name character"></h2>
...
<h2 class="name weapon"></h2>
```
and in your css something like:
```
.name:not( .character) {
```